internal/syncer/connector/asana.go

```
@@ -0,0 +1,1097 @@
+			cancel()
+			if readErr != nil {
+				return readErr
+			}
+			if response.StatusCode >= 400 {
+				lastErr = &asanaAPIError{Status: response.StatusCode, Message: strings.TrimSpace(string(body))}
+				if !isAsanaRetryable(response.StatusCode) {
+					return lastErr
+				}
+			} else {
+				if int64(len(body)) > asanaMaxJSONResponseSize {
+					return fmt.Errorf("Asana API response exceeds maximum size of %d bytes", asanaMaxJSONResponseSize)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)